src/lumigo_tracer/parsers/event_parser.py

```diff
@@ -66,8 +64,10 @@ def is_supported(event) -> bool:
             isinstance(event, Dict)
             and event.get("requestContext")  # noqa
             and event.get("requestContext", {}).get("domainName")  # noqa
+            and event.get("requestContext")  # noqa
+            and event.get("requestContext", {}).get("requestId")  # noqa
```

Comment:
Is this the same logic as above? If so, maybe we should extract it to a function?